terraform/cloudfoundry/network_acls_cf_in.tf

```diff
@@ -123,7 +123,7 @@ resource "aws_network_acl_rule" "100_ssh" {
     rule_number = 100
     rule_action = "allow"
     from_port = 22
-    to_port = 22
+    to_port = 9300
```

Comment:
We're going to have to open up more than we need, but I don't think we should modify this particular rule because:

- the description of the rule is now misleading
- the port range is huge - it makes some of the other rules redundant and covers most of the <1024 privileged ports
- the source address range covers all of the other subnets (including router and cell), rather than just the infra subnets (where the ELBs currently live)

It might be more appropriate to refactor one or both of the `117_kibana` and `118_ingestor_syslog` rules, which apply to the correct source subnet and have higher `from_port` values.